File: westcoast/src/main/java/frc/commands/auto/demo/AVeryMarkCommand.java
package frc.commands.auto.demo;

import edu.wpi.first.wpilibj.Encoder;
import edu.wpi.first.wpilibj.drive.DifferentialDrive;
import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
import frc.maps.*;

public class AVeryMarkCommand extends SequentialCommandGroup {

  public AVeryMarkCommand(
    RobotMap map,
    DifferentialDrive drive,
    Encoder rightEncoder,
    Encoder leftEncoder
  ) {
    super(
    new Square(map, drive, rightEncoder, leftEncoder, 10, 90), 
    new Square(map, drive, rightEncoder, leftEncoder, 10, 90), 
    new Square(map, drive, rightEncoder, leftEncoder, 10, 90), 
    new Square(map, drive, rightEncoder, leftEncoder, 10, 90), 
    new Square(map, drive, rightEncoder, leftEncoder, 10, 90), 
    new Square(map, drive, rightEncoder, leftEncoder, 10, 90), 
    new Square(map, drive, rightEncoder, leftEncoder, 10, 90), 
    new Square(map, drive, rightEncoder, leftEncoder, 10, 90), 
    new Square(map, drive, rightEncoder, leftEncoder, 10, 90), 
    new Square(map, drive, rightEncoder, leftEncoder, 10, 90), 
    new Square(map, drive, rightEncoder, leftEncoder, 10, 90), 
    new Square(map, drive, rightEncoder, leftEncoder, 10, 90)
    );
  }
}

File: westcoast/src/main/kotlin/frc/commands/auto/demo/AVeryMarkCommand.kt
package frc.commands.auto.demo

import edu.wpi.first.wpilibj.Encoder
import edu.wpi.first.wpilibj.drive.DifferentialDrive
import edu.wpi.first.wpilibj2.command.SequentialCommandGroup
import edu.wpi.first.wpilibj2.command.WaitCommand
import frc.commands.auto.demo.Square
import edu.wpi.first.wpilibj2.command.CommandBase
import edu.wpi.first.wpilibj.controller.ProfiledPIDController
import frc.robot.maps.RobotMap

class AVeryMarkCommand(
    map: RobotMap,
    drive: DifferentialDrive,
    rightEncoder: Encoder,
    leftEncoder: Encoder
) : SequentialCommandGroup(
    Square(map, drive, rightEncoder, leftEncoder, 10.0, 90.0),
    Square(map, drive, rightEncoder, leftEncoder, 10.0, 90.0),
    Square(map, drive, rightEncoder, leftEncoder, 10.0, 90.0),
    Square(map, drive, rightEncoder, leftEncoder, 10.0, 90.0),
    Square(map, drive, rightEncoder, leftEncoder, 10.0, 90.0),
    Square(map, drive, rightEncoder, leftEncoder, 10.0, 90.0),
    Square(map, drive, rightEncoder, leftEncoder, 10.0, 90.0),
    Square(map, drive, rightEncoder, leftEncoder, 10.0, 90.0),
    Square(map, drive, rightEncoder, leftEncoder, 10.0, 90.0),
    Square(map, drive, rightEncoder, leftEncoder, 10.0, 90.0),
    Square(map, drive, rightEncoder, leftEncoder, 10.0, 90.0),
    Square(map, drive, rightEncoder, leftEncoder, 10.0, 90.0)
)